app/views/dashboard/index.html.slim

```diff
@@ -0,0 +1,29 @@
+h1.title Your personal dashboard
+
+table.table
+  thead
+    tr
+      th Goal
+      th Monday
+      th Tuesday
+      th Wednesday
+      th Thursday
+      th Friday
+
+  tbody
+    - @goals.each do |goal|
```

Comment:
Hint: Rather than looping over the progresses, you'll want to loop over the past seven days, and then output the progresses for each day.